src/Console/BaseCommand.php

```diff
@@ -45,7 +45,7 @@ abstract class BaseCommand implements CommandInterface
      */
     public function setName(string $name)
     {
-        if (strpos($name, ' ') < 1) {
+        if (!strpos($name, ' ')) {
```

Comment:
```suggestion
        if (!str_contains($name, ' ')) {
```

Isn't this a 'not contains' check?

Reply:
Apparently, there is a special check for starting with a space for some reason.

Reply:
@markstory `!str_contains()` will pass for string with leading space ` foo` which we don't want to allow. There's also a test for that specific case.

Reply:
Of fun.